lib/reversetunnel/localsite.go

```diff
@@ -275,16 +275,17 @@ func (s *localSite) Dial(params reversetunnelclient.DialParams) (net.Conn, error
 	return s.DialTCP(params)
 }
 
-func shouldSendSignedPROXYHeader(signer multiplexer.PROXYHeaderSigner, useTunnel, isAgentlessNode bool, srcAddr, dstAddr net.Addr) bool {
+func shouldSendSignedPROXYHeader(signer multiplexer.PROXYHeaderSigner, useTunnel, isNonTeleportNode bool, srcAddr, dstAddr net.Addr) bool {
```

Comment:
Can we revert the name and the condition? Boolean that is negated as a parameter sounds a bit odd IMO.

Reply:
I could keep the old name name but then should it be maybeSendSignedPROXYHeader `isAgentlessNode := params.IsAgentlessNode || (params.TargetServer == nil && params.ConnType == types.NodeTunnel)`? which can work but looks a bit weird.

I think name `IsNonTeleportNode` is best here, because inverting makes logic harder to follow. It's inversion is not a `TeleportNode` because it combines 2 properties behind it, 'non' part is basically just applied to 'Teleport' part. Fx auth server will have `TeleportNode` = `true` in that case, but it's not a node.